Cargo.toml

```diff
@@ -41,10 +41,10 @@ futures = { version = "0.3.30", default-features = false }
 getrandom = { version = "0.2", default-features = false }
 hex = "0.4.3"
 hkdf = "0.12.3"
-openmls = { git = "https://github.com/xmtp/openmls", rev = "043b347cb18d528647df36f500725ab57c41c7db", default-features = false }
-openmls_basic_credential = { git = "https://github.com/xmtp/openmls", rev = "043b347cb18d528647df36f500725ab57c41c7db" }
-openmls_rust_crypto = { git = "https://github.com/xmtp/openmls", rev = "043b347cb18d528647df36f500725ab57c41c7db" }
-openmls_traits = { git = "https://github.com/xmtp/openmls", rev = "043b347cb18d528647df36f500725ab57c41c7db" }
+openmls = { git = "https://github.com/xmtp/openmls", branch = "coda/expose-some-apis", default-features = false }
+openmls_basic_credential = { git = "https://github.com/xmtp/openmls", branch = "coda/expose-some-apis" }
+openmls_rust_crypto = { git = "https://github.com/xmtp/openmls", branch = "coda/expose-some-apis" }
+openmls_traits = { git = "https://github.com/xmtp/openmls", branch = "coda/expose-some-apis" }
```

Comment:
Lets merge the PR and point to a commit instead 
https://github.com/xmtp/openmls/pull/39#pullrequestreview-2579325061

Also is this something we should commit back to openMLS?

Reply:
Ah yes, good catch. I need to change this back to a rev.
And yes, I think it'd be good to get that into the core openmls. We talked with them during the last meeting about relying on that data from the staged welcome and they said it should be fine.